README.md

```diff
@@ -202,13 +202,17 @@ repeating the mistakes of the past.
 - Write blog posts or speak at meetups and conferences.
 
 #### Collaboration over competition
-- We have a very collaborative culture. There are a number of reasons why this is preferable:
-    - [Competitive cultures tend to not bode well for creating diverse teams.](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a).
+- We have a very collaborative culture. [Competitive cultures tend to not bode well for creating diverse teams](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a), which we value highly.
 
 #### Favour experimentation over argument
 
 #### Be excellent to each other!
 
+#### Inclusion over exclusion
+- We value diversity - and a diverse culture is one that does not exclude people. Bullying,
```

Comment:
First "rule" of the Developer Values - there are are no rules, only guidelines :) https://github.com/mobify/developer-values#what-the-document-is-not

The goal of this document is to describe how we build software, and a big part of that how is how we work with each other. I like the article and would like to keep it because it does a great job at illustrating what kind of effect contemptuous behaviour has. 

Re: "intent matters", I think that could turn into a larger debate than it needs to. The point does not say anything about intent, it simply states that we want to avoid exclusion, and the article is there to articulate in more detail why exclusion hurts teams. Are you advocating that we do not include the link to that article because of the position the author takes on intent?

Btw, definitely happy to consider adding those other links as well though! Just gotta read through them first ;). Other people also had some really good feedback on how we can refine this point, and with all this awesome feedback we should be able to nail this point out of the park! ⚾️🏏